index.js

```diff
@@ -19945,7 +19945,7 @@ function BeeSwarmSimulator(DATA){
 
             arr.push(...MATH.selectFromArray([['starJelly',1],['softWax',MATH.random(1,4)|0],['hardWax',1],['fieldDice',MATH.random(1,4)|0],['smoothDice',MATH.random(1,2)|0],['loadedDice',1],['oil',MATH.random(1,4)|0],['glue',MATH.random(1,4)|0],['neonberry',MATH.random(1,4)|0],['whirligig',MATH.random(1,7)|0],['honeysuckle',MATH.random(1,15)|0],['microConverter',MATH.random(1,4)|0],['jellyBeans',MATH.random(1,6)|0],],Math.min(out.roboChallenge.round/6,3)|0))
 
-            if(out.roboChallenge.round>11&&Math.random()<0.25) arr.push(['purplePotion',1])
+            if(out.roboChallenge.round>11&&Math.random()<0.25) arr.push(['starTreat',1])

```

Comment:
Removed purple potion from the prizes given from the RoboChallenge, replacing it with starTreat